eventsourcing/src/main/java/org/axonframework/eventsourcing/SnapshotterSpanFactory.java

```diff
@@ -0,0 +1,31 @@
+package org.axonframework.eventsourcing;
+
+import org.axonframework.tracing.Span;
+
+/**
+ * Span factory that creates spans for the {@link Snapshotter}.
+ * You can customize the spans of the snapshotter by creating your own implementation.
+ *
+ * @author Mitchell Herrijgers
+ * @since 4.9.0
+ */
+public interface SnapshotterSpanFactory {
+    /**
+     * Creates a new {@link Span} which represents the scheduling of snapshot creation to the snapshotter's executor.
```

Comment:
The word "which" typically comes after a comma. If the sentence doesn't need to be broken up with a comma, you should use "that" instead:
```suggestion
     * Creates a new {@link Span} that represents the scheduling of snapshot creation to the snapshotter's executor.
```

Reply:
I was unsure about this comment, so I decided to look this up. Which vs that has nothing to do with comma's, but with the meaning of the clause; https://www.grammarly.com/blog/which-vs-that/

As the sentence still makes sence without the class (Creates a new Span) I think we should use which here!

Learn something new every day